lib/ckb/wallet.rb

```diff
@@ -82,8 +83,9 @@ def generate_tx(target_address, capacity)
 
     # @param target_address [String]
     # @param capacity [Integer]
-    def send_capacity(target_address, capacity)
-      tx = generate_tx(target_address, capacity)
+    # @param data [String] "0x..."
+    def send_capacity(target_address, capacity, data = "0x")
```

Comment:
Also, the convention in this gem is to use keyword arguments, can we stick to that unified rule?

Reply:
> Also, the convention in this gem is to use keyword arguments, can we stick to that unified rule?

I found that optional keyword arguments are only used in `initialize` method, default parameter value is used in other function, e.g.
https://github.com/nervosnetwork/ckb-sdk-ruby/blob/de883e02a897aa3366666f2e737aa9745efa74d6/lib/ckb/convert_address.rb#L7

Reply:
I see, in this case feel free to leave it as it is.